entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash 
+su
```

Comment:
why do you need this?

Reply:
It uses python installed at `usr/local/bin/python` instead of `opt/venv/bin/python` if there's not an explicit switch to root user. I don't really know why, since there's already `USER root` in the dockerfile, but without it it can't find any of the installed dependencies.

Reply:
Oh, but in any case the entrypoint script is not necessary any more, so I will remove it.